src/react/deckgl.js

```diff
@@ -126,6 +128,18 @@ export default class DeckGL extends React.Component {
       );
     }
 
+    // TODO: add handlers on demand at runtime, not all at once on init
+    this.eventManager = createEventManager(canvas, {})
+    .on({
+      click: this._onClick,
```

Comment:
I assume in the future, we will be generating user intentions on higher level of abstraction instead of calling callbacks at "mousemove" level, right? I mean, deck.gl will distinguishing events that can handle by itself (events lead to viewport change) from those that need to be handled by the app (e. g. events lead to data change). Is my understanding right?